dtfs-central-api/src/helpers/merge-tfm-values-into-portal-deal.ts

```diff
@@ -0,0 +1,14 @@
+import { Deal, DEAL_STATUS, TFM_DEAL_STAGE, TfmDeal } from '@ukef/dtfs2-common';
+
+/**
+ * Updates properties on a portal deal with updates from tfm
+ * @param portalDeal the portal deal returned from the database
+ * @param tfmDeal the tfm deal if it exists
+ * @returns portal `deal` object with relevant fields from tfm-deals added in
+ */
+export const mergeTfmValuesIntoPortalDeal = (portalDeal: Deal, tfmDeal: TfmDeal | null): Deal => {
```

Comment:
It's been a long time - does portal currently consume any other values from TFM? i.e `ukefDecision`, I believe this may come from TFM.

If so, wondering if we should explore a new property which might be clearer than merging status

Could get messy/confusing if another status like "cancelled" is added in the future e.g "reopened" or similar